hw/ip/i2c/doc/_index.md

```diff
@@ -351,7 +351,9 @@ The interrupt `tx_empty` is asserted whenever target intends to transmit data on
 If a target receives a STOP or repeated START signal and there are extra bytes left in TX FIFO, the interrupt `tx_nonempty` is asserted and the FIFO is flushed.
 
 When a host receives enough data from a target, it usually signals the end of the transaction by sending a NACK followed by a STOP or a repeated START.
-In a case when a target receives an ACK and then a STOP/START, the interrupt `ack_stop` is asserted.
+In a case when a target receives a STOP without the prerequisite NACK, the interrupt `unexp_stop` is asserted.
```

Comment:
So I need two cases to test.
ack + stop  and
nack + (no-stop)
is that right?

Reply:
ah i dont think you have to test `nack` + `no stop`.  Since that just means the host isn't doing what it's supposed to.  Eventually that will cause us to just time out I believe.

I think `ack + stop` is more interesting.  Just make sure, like we discussed yesterday, you provide more than enough TX data so that we don't get into a TX stretch state.

Reply:
got it